app/services/food_api/endpoint_helpers.rb

```diff
@@ -0,0 +1,12 @@
+module EndpointHelpers
+  def api_endpoint
+    # The http API endpoint for this service
+    'https://api.nal.usda.gov/fdc/'.freeze
+  end
+
+  # API Endpoint Helpers
+
+  def search_foods_endpoint
+    "v1/foods/search".freeze
+  end
```

Comment:
Generally when we freeze strings we use a constant to store their value. Love the abstraction though!